WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Amount/SimplePaymentsAmount.swift

```diff
@@ -77,7 +79,19 @@ extension SimplePaymentsAmountHostingController: UIAdaptivePresentationControlle
     }
 
     func presentationControllerShouldDismiss(_ presentationController: UIPresentationController) -> Bool {
-        !rootView.viewModel.disableViewActions
+        guard ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) else {
+            return !rootView.viewModel.disableViewActions
+        }
+
+        if viewModel.shouldEnableSwipeToDismiss {
+            return true
+        }
+
+        UIAlertController.presentDiscardChangesActionSheet(viewController: self, onDiscard: { [weak self] in
+            self?.dismiss(animated: true, completion: nil)
+        })
```

Comment:
Same comment as as in `NewOrderHostingController` but making sure we take into account `rootView.viewModel.disableViewActions`. to not present the sheet while a network request is being made.

Reply:
Updated in 2f58a13fb04fa6125a81b04ae22647aaf9e34ef6.

Reply:
I think this needs also to be under the feature flag right? 

If not, in production, this alert would be presented only if the user tries to dismiss the view while a network request is being made. But it will not be presented in a regular dismissal, which would be a bit inconsistent.

Reply:
Good point - updated in 7392ae1fcc40345d5d169708e45c248416ccb5dd.